src/runtime/task/mod.rs

```diff
@@ -89,7 +90,16 @@ pub(crate) struct Task {
     name: Option<String>,
 
     local_storage: StorageMap,
-    pub(super) span: tracing::Span,
+
+    // The `Span` containing the `Task`s `id` and the current step count (if step count recording is enabled)
```

Comment:
I don't understand what it means to say "the Span containing the Task's id" -- could you clarify?

Reply:
?
Sure, but I'm not sure how it can be described differently (in a way which is understandable). This is the span which contains the word "step" (thus the name `step_span`) and which contains the task = id.0 field and optionally the current step count (denoted i = _)